### PR TITLE
[1LP][RFR] remove rest_api fixture from test_rest_services

### PR DESCRIPTION
**Pull request by mkoura**
Replacing the rest_api fixture with appliance.rest_api in test_rest_services.

{{pytest: -v --long-running cfme/tests/services/test_rest_services.py}}

### CODE REVIEW

**Comment by mkoura** [comment]
PRT: one timeout on 5.8, all other passed.

**Comment by mkoura** [comment]
@izapolsk It's timeout issue, I think it's safe to merge. Other variant of this test  (`TestServiceRESTAPI::test_power_service[from_collection]`) succeeded, so I wouldn't increase timeouts further atm. The test is cycling between service power states (on -> off -> on -> suspended). It's maybe too complex, I'll consider splitting it to several tests testing just one transition at a time (in other PR).

File: cfme/tests/services/test_rest_services.py [hunk 1]
```diff
@@ -40,7 +40,7 @@ def a_provider(request):
 def wait_for_vm_power_state(vm, resulting_state):
     wait_for(
         lambda: vm.power_state == resulting_state,
-        num_sec=600, delay=20, fail_func=vm.reload,
+        num_sec=1200, delay=45, fail_func=vm.reload,
```
Comment (mshriver) [inline cfme/tests/services/test_rest_services.py]: **Optional**
Keep the delay lower, save 25 seconds sometimes! Up to you, I don't see much of a problem checking with the same frequency for longer.

**Comment by mshriver** [comment]
Single timeout failure in 58z, upstream failure because of flash message changes in catalog item creation, not part of this PR.

**Pull request merged**